src/DssSpell.sol

```diff
@@ -94,306 +45,8 @@ contract DssSpellAction is DssAction {
-        // D3M Parameter Adjustments Poll: https://vote.makerdao.com/polling/QmWatYqy#poll-detail
-        // Executive Proxy Poll: https://vote.makerdao.com/polling/Qmc9fd3j#poll-detail
-        // Onboard rETH Poll: https://vote.makerdao.com/polling/QmeEV7ph#vote-breakdown (Inside Proxy Spell)
-        // DAI Interest Rate Strategy Poll: https://vote.makerdao.com/polling/QmWodV1J#poll-detail (Inside Proxy Spell)
-        // Forum: https://forum.makerdao.com/t/2023-05-24-spark-protocol-updates/20958
-        DssExecLib.setIlkAutoLineParameters("DIRECT-SPARK-DAI", /* line */ 20 * MILLION, /* gap */ 20 * MILLION, /* ttl */ 8 hours);
-        DssExecLib.authorize(SPARK_PROXY, DssExecLib.esm());
-        ACLManagerLike(SPARK_ACL_MANAGER).addPoolAdmin(SPARK_PROXY);
-        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
-
-        // --- Non-Scope Defined Parameter Adjustments ---
-        // Poll: https://vote.makerdao.com/polling/QmQXhS3Z#poll-detail
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-2-non-scope-defined-parameter-changes-may-2023/20981
-
-        // Increase RETH-A line from 20 million DAI to 50 million DAI
-        // Increase RETH-A gap from 3 million DAI to 5 million DAI
-        DssExecLib.setIlkAutoLineParameters("RETH-A", /* line */ 50 * MILLION, /* gap */ 5 * MILLION, /* ttl */ 8 hours);
-
-        // Increase the RETH-A Stability Fee from 0.75% to 3.74%
-        DssExecLib.setIlkStabilityFee("RETH-A", THREE_PT_SEVEN_FOUR_PCT_RATE, true);
-
-        // Increase the CRVV1ETHSTETH-A Stability Fee from 1.75% to 4.24%
-        DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", FOUR_PT_TWO_FOUR_PCT_RATE, true);
-
-        // Increase the WBTC-A Stability Fee from 4.90% to 5.80%
-        DssExecLib.setIlkStabilityFee("WBTC-A", FIVE_PT_EIGHT_PCT_RATE, true);
-
-        // Increase the WBTC-B Stability Fee from 4.90% to 6.30%
-        DssExecLib.setIlkStabilityFee("WBTC-B", SIX_PT_THREE_PCT_RATE, true);
-
-        // Increase the WBTC-C Stability Fee from 4.90% to 5.55%
-        DssExecLib.setIlkStabilityFee("WBTC-C", FIVE_PT_FIVE_FIVE_PCT_RATE, true);
-
-        // --- RWA015 (BlockTower Andromeda) ---
-        // Poll: https://vote.makerdao.com/polling/QmbudkVR#poll-detail
-        // Forum links:
-        //   - https://forum.makerdao.com/t/mip90-liquid-aaa-structured-credit-money-market-fund/18428
-        //   - https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969
-        //   - https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974
-        _onboardRWA015A();
-        DssExecLib.setChangelogVersion("1.14.13");
-
-        // --- USDP PSM Debt Ceiling ---
-        // Poll: https://vote.makerdao.com/polling/QmQYSLHH#poll-detail
-        // Forum: https://forum.makerdao.com/t/reducing-psm-usdp-a-debt-ceiling/20980
-        // Reduce the PSM-PAX-A Debt Ceiling from 500 million DAI to 0 DAI
 
```

Comment:
```suggestion
```

Nit: spacing (IMO)

Reply:
Won't change. There's always an empty line before `actions()`.

Reply:
Apologies if this is unclear, the line I am referring to here is line 49 (empty space _inside_ `actions()`), which I wass suggesting should be deleted

Please see below:
<img width="346" alt="" src="https://github.com/makerdao/spells-mainnet/assets/103920908/3b2a99ee-eea7-4191-9fe3-36028e4ea95b">

It would look something like this:
```
function actions() public override {
}
```

Precedent for this:
2023-03-08
2023-03-11 (Emergency Spell, my reference)
2023-03-14
2023-03-24

Specifically, I was going back to a previous emergency spell `// Emergency Proposal: Risk and Governance Parameter Changes (11 March 2023)` and just commented the nit based on that.

Resolving as this is style based change and there are less nitt-ish issues + this is an emergency spell so we don't have time to debate these small issues, but for reference the last emergency spell followed the pattern mentioned aboved.